jp2_online/urls.py

```diff
@@ -19,6 +19,8 @@
 urlpatterns = [
     url('', include('base.urls')),
     url(r'^admin/', admin.site.urls),
+    url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     url(r'^administracion/', include('administracion.urls')),
     url(r'^tosp_auth/', include('tosp_auth.urls')),
+    url(r'^perfiles-usuario', include('perfiles_usuario.urls', namespace='perfiles_usuario')),
```

Comment:
We are not using the namespace definition, we are defining the app_name inside the app urls. I saw you already did, so I think this is not needed.